index.html

```diff
@@ -19,7 +19,7 @@
 
     gtag('config', 'G-MFT901EK2W');
   </script>
-  <title>Yuriy's Portfolio</title>
+  <title>Yuriy's Mircroverse Portfolio</title>
 </head>
 
 <body>
```

Comment:
- [ ] There is a contrast issue with your navigation links. I think it could be a good idea to update the hover colors to make your page more accessible :hugs:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/139879623-6a77f1e7-1ebe-41dd-a377-0f09a9b71a75.png)

Reply:
I see the picture you attached and it is very bad. I do have a transition on the navigation links and the transition is on the color and background color. So, the transition, I agree is a little bit long - 2 seconds. For the part of the transition, yeah, it might go through such a bad contrast, but once the transition is finished it will be OK. I am really surprised how you were able to capture this screenshot because if you waited for two seconds you will get this:

![image](https://user-images.githubusercontent.com/8495547/139920939-2dadb690-4324-4ff1-b0a1-5df73cd97841.png)

I will shorten the time of the transition, so the visitor will not have this experience, or it will have it for a really short time, so it will not be noticeable.

Reply:
- [ ] Good job making your animation shorter. This is part of thinking in everyone. Unfortunately, the contrast-ratio is still to low ("2.88"), we can increase this to four by changing it to this shade (#6070ff) or another shade that you find convenient. As long as it gets higher than four.

![image](https://user-images.githubusercontent.com/79114233/139925924-67ec366d-f8b5-4a19-8386-86ad54d83f0a.png)

Reply:
Thanks for noticing it. Yes, these are the original colors stated in the Figma template and I used them as I had to pass my mobile part of the portfolio project. So, now I assume, I have to change them because they are not looking good as the accessibility required. OK. I made the change and have got a 4.57:1 contrast ratio, which Chrome DevTools says is so OK (green).